cspell.yml

```diff
@@ -8,9 +8,16 @@ ignorePaths:
   - tsconfig.json
   - benchmark/github-schema.graphql
   - benchmark/github-schema.json
+  - static/
+  - docusaurus.config.js
 overrides:
-  - filename: '**/docs/APIReference-*.md'
+  - filename: '**/docs-old/APIReference-*.md'
     ignoreRegExpList: ['/href="[^"]*"/']
+  - filename: 'docs/**'
+    words:
+      - 'clsx'
+      - 'infima'
+      - 'stylelint'
```

Comment:
it should be part of `fullstack` dictionary.
https://github.com/streetsidesoftware/cspell-dicts/blob/72d2c43d8b9d4f7af5619618496ae68c69b16908/dictionaries/fullstack/fullstack.txt#L356
Can you please enable it for this dir?